assembly/src/tests.rs

```diff
@@ -37,7 +37,7 @@ fn single_span() {
     let source = "begin push.1 push.2 add end";
     let program = assembler.compile(source).unwrap();
     let expected = "begin span pad incr push(2) add end end";
-    assert_eq!(expected, format!("{}", program));
```

Comment:
These changes are already in https://github.com/0xPolygonMiden/miden-vm/pull/613 and will disappear from the diff here when that PR is landed.